### ISSUE TITLE
the parameters

### ISSUE THREAD

**Issue by Songtao-Wang** [zhysora/PSGan-Family#2]
Hello, I have some questions，
1、What is the value of the max_epochs？
2、When constructing different satellite data sets，are the parameters rasterorigin, pixelwidth, pixelheight of array2raster(newRasterfn, rasterOrigin, pixelWidth, pixelHeight, array, bandSize) different?

**Comment by zhysora** [comment]
Thanks for your comments.
1. "max_epochs" is the training epoch according to your experiment and the dataset.  In our setting, we tend to set it as 50.
2. Although those parameters in different satellites are usually different, we only use the pixel value matrix of the data. So we just set those parameters as some constants.

**Comment by Songtao-Wang** [comment]
OK，Thanks for your reply
1、The image of 128*128*4 size obtained by clipping the Raw image is compared with the corresponding position of the Raw image, 
and it is found that the color of the clipped image is different from the Raw image. Have you ever encountered such a situation.The following figures are an example.
The raw image of GF-1

 

The clipped image&nbsp;&nbsp; 

2、Because tf.train.shuffle_batch is used when loading the data, is it possible for the two training results to be different if all conditions are exactly the same？

------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "zhysora/PSGan-Family"                                                                                    <notifications@github.com&gt;;
发送时间:&nbsp;2021年2月23日(星期二) 上午10:45
收件人:&nbsp;"zhysora/PSGan-Family"<PSGan-Family@noreply.github.com&gt;;
抄送:&nbsp;"寒号鸟"<2876754665@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
主题:&nbsp;Re: [zhysora/PSGan-Family] the parameters (#2)





 
Thanks for your comments.
  
"max_epochs" is the training epoch according to your experiment and the dataset.  In our setting, we tend to set it as 50.
 
Although those parameters in different satellites are usually different, we only use the pixel value matrix of the data. So we just set those parameters as some constants.

**Comment by zhysora** [comment]
1. Clipping may not change the color of the image. Your example here seems to be failed and I can't see it. The reason for the color change may be the setting of your satellite image viewer.
2. Data shuffle is a common operation in training to help the model learn better. Besides this, there are also other random factors that can affect the model. It may be a few random differences between the two same training settings, but quite small. If you want to keep them totally the same, you can try to fix the random seed of TensorFlow.

**Comment by Songtao-Wang** [comment]
OK, thanks for your patience




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "zhysora/PSGan-Family"                                                                                    <notifications@github.com&gt;;
发送时间:&nbsp;2021年2月23日(星期二) 中午11:57
收件人:&nbsp;"zhysora/PSGan-Family"<PSGan-Family@noreply.github.com&gt;;
抄送:&nbsp;"寒号鸟"<2876754665@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
主题:&nbsp;Re: [zhysora/PSGan-Family] the parameters (#2)





  
Clipping may not change the color of the image. Your example here seems to be failed and I can't see it. The reason for the color change may be the setting of your satellite image viewer.
 
Data shuffle is a common operation in training to help the model learn better. Besides this, there are also other random factors that can affect the model. It may be a few random differences between the two same training settings, but quite small. If you want to keep them totally the same, you can try to fix the random seed of TensorFlow.

**Comment by zhysora** [comment]
You are welcome
